qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -390,6 +390,8 @@ def Sawtooth(duration, amp, freq, name):
         "_valid_amp_conditions",
     )
 
+    disable_validation = False
```

Comment:
Since argument is called `constraints`, something like `ignore_constraints` is more intuitive?